pkgs/tools/inputmethods/fcitx-engines/fcitx-libpinyin/datapath.patch

```diff
@@ -0,0 +1,16 @@
+--- a/src/utils.cpp
++++ b/src/utils.cpp
+@@ -9,12 +9,7 @@ char* FcitxLibPinyinGetSysPath(LIBPINYIN_LANGUAGE_TYPE type)
+ #ifdef LIBPINYIN_TOOLS_FOUND
+     if (type == LPLT_Simplified) {
+ #endif
+-        /* portable detect here */
+-        if (getenv("FCITXDIR")) {
```

Comment:
What is the current value of `FCITXDIR`?

Reply:
Thanks for the suggestion. I've done the explanation.